code/obj/item/staffkiosk.dm

```diff
@@ -0,0 +1,287 @@
+	var/cardIcon = "id_clown" //what icon state the card should receive
+	var/tokenIcon = "assist-token"
+	var/dispensedKit = /obj/item/storage/box/staffkit/med //what kit should be dispensed (lo and behold)
+
+/datum/recruitment_role/medical
+	name = "Medical Assistant"
+	accessParent = "Medical Doctor"
+	canAuthorize = list("Medical Doctor","Geneticist","Roboticist","Medical Director","Head of Personnel","Captain")
+	cardIcon = "id_res"
+	tokenIcon = "assist-token-med"
+	dispensedKit = /obj/item/storage/box/staffkit/med
+
+/datum/recruitment_role/robo
+	name = "Robotics Assistant"
+	accessParent = "Roboticist"
+	canAuthorize = list("Roboticist","Medical Director","Head of Personnel","Captain")
+	cardIcon = "id_res"
+	tokenIcon = "assist-token-robo"
+	dispensedKit = /obj/item/storage/box/staffkit/robo
+
+/datum/recruitment_role/engineer
+	name = "Engineering Assistant"
+	accessParent = "Engineer"
+	canAuthorize = list("Engineer","Chief Engineer","Head of Personnel","Captain")
+	cardIcon = "id_eng"
+	tokenIcon = "assist-token-eng"
+	dispensedKit = /obj/item/storage/box/staffkit/eng
+
+/datum/recruitment_role/mechanic
+	name = "Mechanics Assistant"
+	accessParent = "Mechanic"
+	canAuthorize = list("Mechanic","Chief Engineer","Head of Personnel","Captain")
+	cardIcon = "id_eng"
+	tokenIcon = "assist-token-mech"
+	dispensedKit = /obj/item/storage/box/staffkit/mech
+
+/datum/recruitment_role/mining
+	name = "Mining Assistant"
+	accessParent = "Miner"
+	canAuthorize = list("Miner","Chief Engineer","Head of Personnel","Captain")
+	cardIcon = "id_eng"
+	tokenIcon = "assist-token-mine"
+	dispensedKit = /obj/item/storage/box/staffkit/miner
+
+/datum/recruitment_role/sci
+	name = "Research Assistant"
+	accessParent = "Scientist"
+	canAuthorize = list("Scientist","Research Director","Head of Personnel","Captain")
+	cardIcon = "id_res"
+	tokenIcon = "assist-token-sci"
+	dispensedKit = /obj/item/storage/box/staffkit/res
+
+/datum/recruitment_role/chef
+	name = "Culinary Assistant"
+	accessParent = "Chef"
+	canAuthorize = list("Chef","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-chef"
+	dispensedKit = /obj/item/storage/box/staffkit/chef
+
+/datum/recruitment_role/bar
+	name = "Bar Assistant"
+	accessParent = "Bartender"
+	canAuthorize = list("Bartender","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-bar"
+	dispensedKit = /obj/item/storage/box/staffkit/bar
+
+/datum/recruitment_role/botany
+	name = "Botanical Assistant"
+	accessParent = "Botanist"
+	canAuthorize = list("Botanist","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-hyd"
+	dispensedKit = /obj/item/storage/box/staffkit/botany
+
+
+//equipment kits
+
+/obj/item/storage/box/staffkit/med
+	name = "medical assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
```

Comment:
`/obj/item/paper/book/pocketguide/medical`?

Reply:
Swapping that in place of the bandage.